week-8/Homework/mandatory/3-dog-photo-gallery/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Dog Photo Galary</title>
+  </head>
+  <body>
+    <div>
+      <button id="getADog">click to be happy</button>
+      <ul id="dogList"></ul>
+    </div>
+    <script src="script.js"></script>
```

Comment:
Next time, try to take advantage of window.onload event. Then you don't have to worry about where you load the script as the code will be initialized when browser loaded everything, in other words, the onload listener is executed when the page has been fully loaded along with all resources.

Reply:
I will try to remember implementing `"use strict";` and `window.onload = function();` on all my future code.